### PR TITLE
Kudos identity rebased with master

### PR DESCRIPTION
**Pull request by saneshark**
- include bug fix for MFA
- verified that merge_attributes on app/models/user would correctly set the updated MFA, but I should probably look to add an E2E spec for this once we're ready using the the integration_recorder.rb stuff I introduced before.

### CODE REVIEW

**Comment by saneshark** [comment]
@patrickvinograd for convenience here is diff of actual change since you last looked: https://github.com/department-of-veterans-affairs/vets-api/pull/1391/commits/5b6eee729b9c7a3f8a8690a697822fe926d249e6

File: lib/saml/user_attributes/dslogon.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      def dslogon_assurance
+        attributes['dslogon_assurance']
+      end
+
+      def loa
+        { current: loa_current, highest: loa_highest }
+      end
+
+      def multifactor
+        attributes['dslogon_multifactor']
```
Comment (patrickvinograd) [inline lib/saml/user_attributes/dslogon.rb]: this should be `attributes['multifactor']` - see https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4965

File: lib/saml/user_attributes/mhv.rb [hunk 2]
```diff
@@ -2,36 +2,85 @@
...
+      end
+
+      # NOTE: This attribute is originated by id.me
+      def email
+        attributes['email']
+      end
+
+      # NOTE: This attribute is originated by id.me
+      def multifactor
+        attributes['mhv_multifactor']
```
Comment (patrickvinograd) [inline lib/saml/user_attributes/mhv.rb]: Guessing this should also just be `attributes['multifactor']` - i.e. multifactor is not a mhv/dslogon-related attribute, so it doesn't get prefixed.

Comment (patrickvinograd) [reply lib/saml/user_attributes/mhv.rb]: I haven't confirmed that for MHV but have for DSLogon, I would bet a large sum that mhv is the same though.

Comment (saneshark) [reply lib/saml/user_attributes/mhv.rb]: you're probably right. Will change this too and we can verify when we're testing mhv.

**Pull request merged**